### PR TITLE
[Streaming][Minor]Fix some error docs in streaming examples

### PR DESCRIPTION
**Pull request by jerryshao**
Small changes, please help to review, thanks a lot.

### CODE REVIEW

File: examples/src/main/python/streaming/kafka_wordcount.py [hunk 1]
```diff
@@ -23,7 +23,7 @@
  http://kafka.apache.org/documentation.html#quickstart
 
  and then run the example
-    `$ bin/spark-submit --driver-class-path external/kafka-assembly/target/scala-*/\
+    `$ bin/spark-submit --jars external/kafka-assembly/target/scala-*/\
```
Comment (srowen) [inline examples/src/main/python/streaming/kafka_wordcount.py]: Am I right that this is the right example for 1.3+ only?

Comment (jerryshao) [inline examples/src/main/python/streaming/kafka_wordcount.py]: Yes, python Kafka API is added in 1.3 and using `--driver-class-path` is not correct, need to change to `--jars`.

Comment (srowen) [inline examples/src/main/python/streaming/kafka_wordcount.py]: OK, so this should be merged for 1.3+, but not 1.2?

Comment (jerryshao) [inline examples/src/main/python/streaming/kafka_wordcount.py]: Yeah, exactly!

**Pull request closed without merging**